ezgui/src/shaders/vertex_300.glsl

```diff
@@ -18,8 +18,8 @@ void main() {
     float screen_x = (position[0] * transform[2]) - transform[0];
     float screen_y = (position[1] * transform[2]) - transform[1];
     // Translate that to clip-space or whatever it's called
-    float x = (screen_x / window[0] * 2.0) - 1.0;
-    float y = (screen_y / window[1] * 2.0) - 1.0;
+    float x = (screen_x / window[0]) - 1.0;
```

Comment:
This might be wrong/controversial. I know nothing about shader programming. Only that without this change, the graphics look "too big" and the winit events (like mouse hover/clicks) don't correspond with what's rendered on the screen.

Reply:
Weird, I wonder why it's different here. Fine for now. I don't understand much of shaders either; I figured out lots of this through trial-and-error. https://learnopengl.com/Getting-started/Coordinate-Systems is a good reference; I believe clip-space needs to be [-1, 1]. I think the effects of this change:
- `screen_y=0`: still `-1`
- `screen_y=half of window height`: from `0` to `-0.5`
- `screen_y=window height`: from `1` to `0`
I'm not sure that's right, but we can iterate on it in a followup.

Reply:
FWIW, I tried reverting this, and wasm now works again normally (except for resizing the browser window). So I think something else needs to divide by scale factor for wasm.